### ISSUE TITLE
BotW won't load 2nd time after using transferable shader cache

### ISSUE THREAD

**Issue by goozchos** [cemu-project/Cemu#837]
Running 2.0-39 on MacOS.

When using pre-compiled shaders for BotW, it will load them all fine the first time. The "compiling cached shaders" (8600+) and "loading cached Vulkan pipelines" (37000+) progress messages come up when loading the game and it will play without any shader compilation stutter, but if I quit cemu and open it again, it will crash when loading the game the second time. 

If I delete the shaderCache folder and put the pre-compiled shaders there again, it will load them fine again and play the game, but only once. If i close cemu and open it again the same thing happens.

Log: https://pastebin.com/rwkxZmEX

**Comment by goeiecool9999** [comment]
This is a duplicate of #396

**Comment by goozchos** [comment]
> This is a duplicate of #396

Got it, thanks. I didn't find that through my initial Google search. I'll try the workarounds listed there. Hopefully that one will get fixed sometime soon.
